app/src/main/java/org/fossasia/openevent/data/Session.java

```diff
@@ -61,117 +61,101 @@ public Session(int id, String title, String subtitle,
         this.microlocations = microlocations;
     }
 
-    public int getMicrolocations() {
-        return microlocations;
-    }
-
-    public void setMicrolocations(int microlocations) {
-        this.microlocations = microlocations;
-    }
-
     public int getId() {
         return id;
     }
 
-
     public void setId(int id) {
         this.id = id;
     }
 
-    public int[] getSpeakers() {
-        return speakers;
-    }
-
-    public void setSpeakers(int[] speakers) {
-        this.speakers = speakers;
-    }
-
     public String getTitle() {
-        return escapeChar(title);
+        return title;
     }
 
     public void setTitle(String title) {
         this.title = title;
     }
 
-
     public String getSubtitle() {
-        return escapeChar(subtitle);
+        return subtitle;
     }
 
-
     public void setSubtitle(String subtitle) {
         this.subtitle = subtitle;
     }
 
-
     public String getSummary() {
-        return escapeChar(summary);
+        return summary;
     }
 
     public void setSummary(String summary) {
         this.summary = summary;
     }
 
-
     public String getDescription() {
-        return escapeChar(description);
+        return description;
     }
 
-
     public void setDescription(String description) {
         this.description = description;
     }
 
-
     public String getStartTime() {
-        return escapeChar(startTime);
+        return startTime;
     }
 
-
     public void setStartTime(String startTime) {
         this.startTime = startTime;
     }
 
-
     public String getEndTime() {
-        return escapeChar(endTime);
+        return endTime;
     }
 
-
     public void setEndTime(String endTime) {
         this.endTime = endTime;
     }
 
-
     public String getType() {
-        return escapeChar(type);
+        return type;
```

Comment:
Yeah instead of using the escapeChar function, I used Sql Escape string, so I removed this.